xarray/util/generate_aggregations.py

```diff
@@ -427,6 +426,9 @@ def generate_code(self, method):
     Method("std", extra_kwargs=(skipna, ddof), numeric_only=True),
     Method("var", extra_kwargs=(skipna, ddof), numeric_only=True),
     Method("median", extra_kwargs=(skipna,), numeric_only=True),
+    # Cumulatives:
+    Method("cumsum", extra_kwargs=(skipna,)),
+    Method("cumprod", extra_kwargs=(skipna,)),
```

Comment:
Note we previously had `numeric_only=True` though I guess it may not be needed here.

Reply:
I added it now, or should I revert?

Reply:
LGTM. Let's open an issue to add tests when we relax it.